contracts/sol6/wrappers/WrapReadTokenDataV2.sol

```diff
@@ -0,0 +1,82 @@
+pragma solidity 0.6.6;
+
+import "@kyber.network/utils-sc/contracts/IERC20Ext.sol";
+import "@kyber.network/utils-sc/contracts/Withdrawable.sol";
+
+interface IConversionRatesGetSteps {
+    function getStepFunctionData(IERC20Ext token, uint command, uint param)
+        external view returns(int);
+}
+
+contract WrapReadTokenDataV2 is Withdrawable {
+    constructor(address _admin) public Withdrawable(_admin) {}
+
+    function readQtyStepFunctions(
+        IConversionRatesGetSteps rate,
+        IERC20Ext token
+    )
+        external view
+        returns (
+            int numBuyRateQtyStepsX, 
```

Comment:
not sure why we need to return numBuyRateQtyStepsX

Reply:
I mean ether we not return numBuyRateQtyStepsX or return as previous version.

Reply:
In general, is it necessary to know what the num of steps are? Since it's simply the array length of the argument after, right?

Reply:
Also, numSteps can be uint right (see related comment below)